docs.css

```diff
@@ -238,8 +239,19 @@ header button {
   padding: 0.5em 0;
 }
 
+.component-list-item:focus {
+  background-color: black;
+  color: white;
+}
+
 .component-list-item:hover {
-  background-color: #1877f2;
+  background-color: black;
+  color: white;
+  transition: 0.5s ease;
+}
+
+.component-list-item:active {
+  background-color: black;
   color: white;
```

Comment:
Here you can use color variables. As I can see you are using it consistently, seems like you missed it here.